src/__stories__/Forms/Input.tsx

```diff
@@ -38,6 +38,6 @@ storiesOf('Forms:Input', module)
     <Input
       {...inputKnobs()}
       value={store.get('value')}
-      onChange={event => store.set({ value: event.currentTarget.value })}
+      onChange={(event: any) => store.set({ value: event.currentTarget.value })}
```

Comment:
Why is any needed here now?

Reply:
I don't know! And it drives me nuts! Storybook just started to complain about this once I wrapped `Input` in a `React.forwardRef`.

There were multiple cases where I had to change an `event` to an `event: any` and each error only started happening _after_ I wrapped that particular component with a `React.forwardRef` which makes me think that wrapping them subtly changes the type.

Which if true, technically means that this PR is a BREAKING CHANGE because code that currently compiles with ui-kit _won't_ if you upgrade ui-kit because it'll start complaining about implicit `any`s? I DON'T UNDERSTAND WHAT IS HAPPENING 😱 

So I'm just shrugging it off? Mostly harmless right?

Reply:
> Which if true, technically means that this PR is a BREAKING CHANGE because code that currently compiles with ui-kit _won't_ if you upgrade ui-kit because it'll start complaining about implicit `any`s? I DON'T UNDERSTAND WHAT IS HAPPENING

Yeah, that's what I am afraid of.

Technically the return type is indeed a bit different, but don't get why it affects event handlers (and most likely other stuff).

Reply:
this is the kind of TS "experience" that makes me worried that restricting the types from HTMLOrSVGElement to something more specific might mean we have to do weird casting and things.

Reply:
@wmhilton 
Actually, my "TS experience"  is "spend 2387283782 hours finding a workaround for some dumb typing problem" and eventually don't gain anything, because the code is very simple regardless.